SPECS/kernel/kernel.spec

```diff
@@ -29,7 +29,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        6.6.2.1
-Release:        3%{?dist}
+Release:        4%{?dist}
```

Comment:
Due to some current entangled kernel-related specs, `kernel-signed`, `kernel-headers`, and corresponding manifest .txt files need to also be updated (i.e., https://github.com/microsoft/CBL-Mariner/commit/b6646353c8ddda6768cfa5399eaf1865bf67c86e)

Reply:
ack, updated all of them to match, thanks